ndicapimodule.cxx

```diff
@@ -961,13 +991,30 @@ static PyObject* Py_ndiGetBXPortStatus(PyObject* module, PyObject* args)
   if (PyArg_ParseTuple(args, "O&c:plGetBXPortStatus",
                        &_ndiConverter, &pol, &port))
   {
-    result = ndiGetGXPortStatus(pol, port);
+    result = ndiGetBXPortStatus(pol, port);
```

Comment:
I am not familiar with the GX calls, but is this correct?

Reply:
I'm not sure. I haven't used Py_ndiGetBXPortStatus in my application. I noticed when I was doing a find and replace that  Py_ndiGetBXPortStatus called ndiGetGXPortStatus, which looked like a typo to me. We should check with @eruffaldi and I'll test it on one of my trackers.

Reply:
I just checked using our Polaris Vega. I think it was a bug, i.e. Py_ndiGetBXPortStatus always returns zero for me without this change, with this change I get an appropriate return value (49).

Reply:
*sigh* Sorry, that's my fault. Github diff shrunk the code between ndiGetGX and ndiGetBX so I thought you were changing GX to BX... disregard all of this.